outputs.tf

```diff
@@ -0,0 +1,8 @@
+# Copyright 2018 Google LLC. This software is provided as-is,
+# without warranty or representation for any use or purpose.
+# Your use of it is subject to your agreements with Google.
+
+output "content" {
+  description = "startup-script-stdlib.sh content as a string value."
+  value       = "${file("${path.module}/files/startup-script-stdlib.sh")}"
```

Comment:
Nitpick: Our internal standards guide recommends that scripts called by Terraform should be kept in `scripts/`. You can follow this if you like, otherwise feel free to resolve this comment.

Reply:
I thought about this too, and it was originally in scripts/.  At the Terraform layer, the file is treated as a big string blob and passed as a resource attribute, it's never executed by Terraform directly.  Given this use case, we moved it into the files/ directory.